Domains/0-Core/BisCore.remarks.md

```diff
@@ -553,3 +553,17 @@ An Entity is modeled as a `bis:RoleElement` when a set of external circumstances
 > Behavior: The system handler (C++) for `RoleElement` will only permit instances to be inserted into a `RoleModel`.
 This behavior applies to all `RoleElement` subclasses.
 `FunctionalElement` (from the `Functional` schema) is the most widely known subclass of `RoleElement`.
+
+## Relationship Classes
+
+### PhysicalTypeHasAdditionalRecipes
+
+The `RecipeDefinitionElement` already associated to a `TypeDefinition` via its _Recipe_ navigation-property shall not be also included in the associations captured by `PhysicalTypeHasAdditionalRecipes` instances.
```

Comment:
I think we need to include a diagram of how we intend to use this.  Maybe we can use the 3d plan recipe.